index.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 /* eslint no-var: 0 */
 var once = require('once');
+
 var setups;
 
 exports.setups = function (fns) {
-	setups = fns.map(function (fn) {
+	setups = fns.map(fn => {
```

Comment:
so in the new version of node, how do you get rid of errors when try to run the project?